pkg/exporters/stackdriver/config/config.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/node-problem-detector/pkg/exporters/stackdriver/metadata"
+)
+
+var (
+	defaultExportPeriodString         = (60 * time.Second).String()
+	defaultEndpoint                   = "monitoring.googleapis.com:443"
+	defaultMetadataFetchTimeoutString = (600 * time.Second).String()
+)
+
+type StackdriverExporterConfig struct {
+	ExportPeriodString          string            `json:"exportPeriod"`
```

Comment:
I think if you use yaml instead of json, it can unmarshal `10s` to `time.Duration` automatically. Then you do not need a `XYZString` field.

Reply:
Thanks for the tips!
Since all other config files are in json format, I think it's safer to keep it like that. Do you think it's worth it to switch to yaml?

Reply:
I don't think that's a problem myself. Maybe ping @Random-Liu to check if there is any history or it does not matter?

Reply:
Done. Currently only storing a string formatted `ExportPeriod`, and parses it at run time.

Filed #344 for yaml support, which will be done in a separate PR.